velox/docs/monthly-updates/february-2024.rst

```diff
@@ -0,0 +1,68 @@
+********************
+February 2024 Update
+********************
+
+Core Library
+============
+
+* Add support for aggregations over distinct inputs to StreamingAggregation.
+* Add stats for null skew in join operator.
+* Add background profiler that starts Linux perf on the Velox process.
+* Fix ``out of range in dynamic array`` error in Task::toJson.
+* Delete unused ``max_arbitrary_buffer_size`` config.
+
+Presto Functions
+================
+
+* Add :func:`typeof`, :func:`from_iso8601_date` scalar functions.
+* Add support for deserializing a single column in Presto page format.
```

Comment:
This item doesn't belong to "Presto Functions" section. Let's move it to "Core Library".